python/state_space_estimation/estimation.py

```diff
@@ -127,14 +130,38 @@ def choose_states(self, n_states, alpha=0.05, tests=['score', 'constraint'], ret
                 results['valid'] = results['valid'].astype(bool)
         return results
 
-            
-    def choose_states_parallel(self, n_states, alpha=0.05, tests=['score', 'constraint'], return_tests=True, verbose=False):
-        '''
-        See self.choose_states; implements same functionality with a parallel backend.
-        '''
-        variables = self.data.columns.values[:np.int64(len(self.data.columns.values)/2)]
+    def choose_states_parallel(self, n_states,
+                               alpha=0.05,
+                               tests=['score', 'constraint'],
+                               return_tests=True,
+                               serial=False,
+                               verbose=False):
+        """
+        Args:
+            n_states:
+            serial: bool: whether to perform the iterations more slowly, not in parallel
+
+        Returns:
+             See self.choose_states; implements same functionality with a parallel backend.
+        """

```

Comment:
Need `opts` to protect against reordering of the options `in self.evaluate_states()`